### PR TITLE
update policy for mate to access elb and additional command line flags 

### PR DESCRIPTION
**Pull request by ideahitme**
@linki

### CODE REVIEW

File: cluster/userdata-master.yaml [hunk 1]
```diff
@@ -780,28 +780,32 @@ write_files:
...
+              - name: AWS_REGION
+                value: {{REGION}}
               args:
               - --producer=kubernetes
               - --kubernetes-domain={{HOSTED_ZONE}}.
               - --consumer=aws
               - --aws-hosted-zone={{HOSTED_ZONE}}.
-              - --no-sync
+              - --sync
+              - --aws-record-group-id={{WEBHOOK_CLUSTER_NAME}}
```
Comment (linki) [inline cluster/userdata-master.yaml]: let's keep the `aws-` prefixed flags together

Comment (ideahitme) [inline cluster/userdata-master.yaml]: done

**Comment by linki** [comment]
👍

**Comment by ideahitme** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

File: cluster/userdata-master.yaml [hunk 2]
```diff
@@ -780,28 +780,32 @@ write_files:
...
+              version: v0.0.6
             annotations:
               scheduler.alpha.kubernetes.io/critical-pod: ''
               scheduler.alpha.kubernetes.io/tolerations: '[{"key":"CriticalAddonsOnly", "operator":"Exists"}]'
               iam.amazonaws.com/role: {{STACK_NAME}}-{{STACK_VERSION}}-app-mate
           spec:
             containers:
             - name: mate
-              image: registry.opensource.zalan.do/teapot/mate:v0.0.5
+              image: registry.opensource.zalan.do/teapot/mate:v0.0.6
```
Comment (mikkeloscar) [inline cluster/userdata-master.yaml]: It's pushed to the registry now :)

**Pull request merged**